wp-content/themes/reconasia/assets/_scss/pages/archive.scss

```diff
@@ -1,6 +1,96 @@
 @use '../abstracts' as *;
 @use '../components/entry-header';
 
+.archive {
+  &__featured {
+    position: relative;
+    margin: 0 calc(var(--container-padding) * -1);
+    padding: 0 var(--container-padding);
+    background: $color-bg-dark-200;
+    @include shadow('lg');
+
+    @include breakpoint('small') {
+      margin: 0 auto;
+      padding: 0;
+    }
+
+    @include breakpoint('medium') {
+      background: transparent;
+      box-shadow: none;
+    }
+
+    &::after {
+      content: '';
+      position: absolute;
+      top: -20px;
+      right: calc(var(--container-padding) * -1);
+      width: calc(100% - var(--container-padding));
+      height: 50px;
+      background: #fff;
+    }
+
+    &-label {
+      position: relative;
```

Comment:
Needs a font-weight of 400

![image](https://user-images.githubusercontent.com/45780666/113622560-2a28a900-962b-11eb-917d-cfe9831efe16.png)